packages/app-project/package.json

```diff
@@ -26,6 +26,7 @@
     "babel-plugin-dynamic-import-node": "~2.2.0",
     "counterpart": "~0.18.6",
     "d3": "~5.9.1",
+    "graphql-request": "^1.8.2",
```

Comment:
Could this be changed to tilde? I didn't find a good way to make that the default between npm and lerna, so we just have to remember to do that.

Reply:
Sure.